server/server.go

```diff
@@ -415,7 +415,7 @@ type queueState struct {
 	lastSent  uint64
 	subs      []*subState
 	shadow    *subState // For durable case, when last member leaves and group is not closed.
-	stalled   bool
+	stalled   int       // number of stalled members
```

Comment:
recommend making this variable name more descriptive `stalledMemberCnt` or `stalledSubCnt`. parsing some of the latter logic of subscriber stalled flag vs. this stalled var becomes hard to follow.